packages/core/src/tech/HlsJsTech.ts

```diff
@@ -1,14 +1,14 @@
 import BaseTech, {
-  IBaseTechOptions,
   IVideoLevel,
   PlaybackState,
 } from './BaseTech';
 import Hls from 'hls.js';
 import { PlayerEvent } from '../util/constants';
+import { IWebPlayerOptions } from '../WebPlayer';
 
 const DEFAULT_CONFIG = {
   capLevelOnFPSDrop: true,
-  capLevelToPlayerSize: true,
+  capLevelToPlayerSize: false,
```

Comment:
Changed default behaviour - might be something to discuss?

Reply:
Just thinking if the previous default is relevant to when used as web-component?

Reply:
I mean from a bandwidth optimization it is always relevant to not play out higher resolution than what can be displayed due to the element's pixel size.

Maybe change it the other way around then - `disablePlayerSizeLevelCap`?

Reply:
Yes, sounds great!

Reply:
Perhaps expose that option in the web-component while at it?

Reply:
Absolutely - and I'll update the Readme as well.